### ISSUE TITLE
How to fail gitlab pipeline when danger throws fail messages

### ISSUE THREAD

**Issue by khalaniaiyaj** [danger/dart#31]

**Comment by HelloCore** [comment]
@khalaniaiyaj Hello, Thank you for reporting this issue.


I've just published `danger_dart` 1.0.1, the new version contains the argument `--failOnErrors`, so you can use it to fail the pipeline.

To update to the new version, you will need to run pub activate again.
```
$ pub global activate danger_dart
```

To use this argument.
```
$ danger_dart ci --failOnErrors
```

**Comment by khalaniaiyaj** [comment]
Thank you brother
